forge-gui/res/cardsfolder/upcoming/micromancer.txt

```diff
@@ -0,0 +1,8 @@
+Name:Micromancer
+ManaCost:3 U
+Types:Creature Human Wizard
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChange | OptionalDecider$ You | TriggerDescription$ When CARDNAME enters the battlefield, you may search your library for an instant or sorcery card with mana value 1, reveal it, put it into your hand, then shuffle.
+SVar:TrigChange:DB$ ChangeZone | Origin$ Library | Destination$ Hand | ChangeType$ Instant.cmcEQ1,Sorcery.cmcEQ1 | ChangeNum$ 1
+DeckNeeds:Type$Instant|Sorcery
+Oracle:When Micromancer enters the battlefield, you may search your library for an instant or sorcery card with mana value 1, reveal it, put it into your hand, then shuffle.
```

Comment:
Hmm, is the cmcEQ1 enough to warrant `AI:RemoveDeck:Random` ? It's a pretty narrow condition for this card to be useful...